lib/ansible/cli/galaxy.py

```diff
@@ -301,6 +302,10 @@ def add_list_options(self, parser, parents=None):
 
         list_parser.add_argument(galaxy_type, help=galaxy_type.capitalize(), nargs='?', metavar=galaxy_type)
 
+        if galaxy_type == 'collection':
+            list_parser.add_argument('-o', '--output', dest='output_format',
```

Comment:
might be a good idea to have a 3rd option like `default` or `human` which is the current behaviour. You can also have a `choices=[]` kwarg where you define the valid choices. This helps both on validation and people using argcomplete to tab complete the options.